Event application.js

```diff
@@ -0,0 +1,10 @@
+// Event application
+function getEventWeekday(eventDay) {
+    const daysOfWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+    const today = new Date();
+    const day = today.getDay();
+    const eventWeekday = (day + eventDay) % 7;
```

Comment:
Well done, this works well! Just as an extra exercise, how would you cover cases when the `eventDay` is negative?